static/src/javascripts/projects/common/modules/commercial/liveblog-dynamic-adverts.js

```diff
@@ -0,0 +1,101 @@
+define([
+    'bonzo',
+    'common/utils/fastdom-promise',
+    'common/utils/config',
+    'common/utils/mediator',
+    'common/modules/commercial/create-ad-slot',
+    'common/modules/article/space-filler',
+    'common/modules/commercial/dfp-api'
+], function (
+    bonzo,
+    fastdom,
+    config,
+    mediator,
+    createAdSlot,
+    spaceFiller,
+    dfp
+) {
+    var INTERVAL = 5;      // number of posts between ads
```

Comment:
Technically, the house style is to only use on var declaration. I'm not a huge fan of that rule, though...